src/routes.js

```diff
@@ -513,6 +513,18 @@ const routes = [
             project: () => project.permits('dataset.list')
           }
         }
+      }),
+      asyncRoute({
```

Comment:
test things like 
- title (title meta field)
- if you're a data collector, you can't visit this page (validateData) 
- going from overview to data tab preserves project data (preserveData)
- login loop (requireLogin)